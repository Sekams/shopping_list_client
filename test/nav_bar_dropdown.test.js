import React from 'react';
import NavBarDropdown from '../src/components/nav_bar_dropdown'

test("renders the nav_bar_dropdown component", () => {
    const wrapper = shallow(
        <NavBarDropdown />
    );
    expect(wrapper).toMatchSnapshot();
});