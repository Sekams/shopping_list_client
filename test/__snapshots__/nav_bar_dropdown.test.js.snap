// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the nav_bar_dropdown component 1`] = `""`;
